apps/finance/app/src/components/Transfers.js

```diff
@@ -161,143 +158,184 @@ const Transfers = React.memo(({ dao, tokens, transactions }) => {
+              transferTypeFilter={selectedTransferType}
+              onTransferTypeChange={handleTransferTypeChange}
+              compactMode={compactMode}
+              symbols={symbols}
+              transferTypes={TRANSFER_TYPES_STRING}
+            />
           )}
-        </div>
+          {emptyResultsViaFilters && (
+            <EmptyFilteredTransfers onClear={handleClearFilters} />
+          )}
+        </React.Fragment>
+      }
+      fields={
+        emptyResultsViaFilters
+          ? []
+          : [
+              { label: 'Date', priority: 2 },
+              { label: 'Source/recipient', priority: 3 },
+              { label: 'Reference', priority: 1 },
+              { label: 'Amount', priority: 2 },
+            ]
+      }
+      entries={filteredTransfers.sort(
+        ({ date: dateLeft }, { date: dateRight }) =>
+          // Sort by date descending
+          compareDesc(dateLeft, dateRight)
+      )}
+      renderEntry={({
+        date,
+        entity,
+        reference,
+        isIncoming,
+        numData: { amount },
+        token,
+      }) => {
+        const formattedDate = format(date, "yyyy-MM-dd'T'HH:mm:ss.SSSxxx")
+        const { symbol, decimals } = tokenDetails[toChecksumAddress(token)]
+        const formattedAmount = formatTokenAmount(
+          amount,
+          isIncoming,
+          decimals,
+          true,
+          { rounding: 5 }
+        )
+
+        return [
+          <time dateTime={formattedDate} title={formattedDate}>
+            {format(date, 'dd/MM/yy')}
+          </time>,
+          <div
+            css={`
+              padding: 0 ${0.5 * GU}px;
+              ${above('medium') &&
+                `
+                  display: inline-flex;
+                  max-width: ${above('large') ? 'unset' : '150px'};
+                `}
+            `}
+          >
+            <LocalIdentityBadge
+              networkType={network.type}
+              entity={entity}
+              address={entity}
+            />
+          </div>,
+          <div
+            css={`
+              padding: 0 ${0.5 * GU}px;
+            `}
+          >
+            {reference}
+          </div>,
+          <span
+            css={`
+              font-weight: 600;
+              color: ${isIncoming ? newTheme.positive : newTheme.negative};
+            `}
+          >
+            {formattedAmount} {symbol}
+          </span>,
+        ]
+      }}
+      renderSelectionCount={count =>
+        `${count} transfer${count !== 1 ? 's' : ''} selected`
+      }
+      renderEntryActions={({ entity, transactionHash }) => (
+        <ContextMenu>
+          <ContextMenuViewTransaction
+            transactionHash={transactionHash}
+            network={network}
+          />
+          <ContextMenuItemCustomLabel entity={entity} />
+        </ContextMenu>
       )}
-    </section>
+    />
   )
 })
 
 Transfers.propTypes = {
-  dao: PropTypes.string.isRequired,
+  dao: PropTypes.string,
```

Comment:
Technically this is still required, but I see how you would get errors because we're missing an empty state to avoid rending `Transfers` if not enough information is available.